src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+img {
+  margin: 10px 50px;
+  height: 40px;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  justify-content: space-between;
+  height: 60px;
+  padding-right: 50px;
+}
+
+nav {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+  height: 60px;
+}
+
+ul {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  flex-wrap: nowrap;
+}
+
+li {
+  position: relative;
+  margin: 0 20px 0 0;
+  padding: 0;
+  justify-content: space-between;
+}
+
+a {
+  display: inline-block;
+  text-align: center;
+  text-decoration: none;
+  color: #000;
+  height: 100%;
+  line-height: 60px;
+  position: relative;
+}
+
+.line a::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  opacity: 1;
+}
+
+.navItem.line {
+  color: #00acdc;
+}
+
+a:hover,
+.line a {
+  color: #00acdc;
+}
+
+.firstch {
```

Comment:
Consider using `:first-child`, `:last-child` to set some depending on this styles